l10n_be_iso20022_pain/models/account_payment_line.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import re
+
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+
+class AccountPaymentLine(models.Model):
+    _inherit = 'account.payment.line'
+
+    communication_type = fields.Selection(
+        selection_add=[
+            ('bba', 'BBA Structured Communication'),
```

Comment:
@ThomasBinsfeld s/'BBA Structured Communication'/_('BBA Structured Communication') so we can translate it laters

Reply:
Thanks, you're right. I thought it was not required in selection_add.

Reply:
Done ! And I added french translation.